### DISCUSSION TITLE
Stand up Tuesday May 23rd

### DISCUSSION THREAD

**Discussion by CristianCazares** [Bugs-io/TC3001B#82] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- Developed a KNN analysis on the certifications based on the similarity of their names.
- Started working on `Más cursadas` graph

**What are your priorities for the next 24 hours?**:
- Continue working on `Más cursadas` graph

**Blockers**:
- lack of will

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
- Worked on aux function for sending data to analysis service

**What are your priorities for the next 24 hours?**:
- Finished that implementation

**Blockers**:
- None

**Shoutouts**:
- @CristianCazares for doing barbell press

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Tried to add firstName and lastName
- Found some (more) bugs regarding login

**What are your priorities for the next 24 hours?**:
- Solve said bugs
- Whitelist our vercel deployments from the CORS Middleware
- Add firstName and lastName

**Blockers**:
- DB connection problems

**Shoutouts**:
- @BrandonMagana for a great gym sesh

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Started working on plotting the rest of the graphs

**What are your priorities for the next 24 hours?**:
- Plot the rest of the graphs

**Blockers**:
- Estuvo cabron el día 

**Shoutouts**:
- @todos
